src/svc_auth_gss.c

```diff
@@ -468,13 +493,17 @@ svcauth_gss_validate(struct svc_req *req,
 	gss_buffer_desc rpcbuf, checksum;
 	OM_uint32 maj_stat, min_stat, qop_state;
 	u_char rpchdr[RPCHDR_LEN];
+	struct timespec start_time;
```

Comment:
I'd rather have start_time conditionally compiled out so it doesn't upset static code analysis when metrics are compiled out. But not sure how to do this with the observe function...